demo-app/src/commonMain/kotlin/dev/sargunv/maplibrecompose/demoapp/demos/LocalTilesDemo.kt

```diff
@@ -0,0 +1,67 @@
+package dev.sargunv.maplibrecompose.demoapp.demos
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import dev.sargunv.maplibrecompose.compose.MaplibreMap
+import dev.sargunv.maplibrecompose.compose.layer.RasterLayer
+import dev.sargunv.maplibrecompose.compose.rememberCameraState
+import dev.sargunv.maplibrecompose.compose.rememberStyleState
+import dev.sargunv.maplibrecompose.compose.source.rememberRasterSource
+import dev.sargunv.maplibrecompose.core.CameraPosition
+import dev.sargunv.maplibrecompose.core.source.TileSetOptions
+import dev.sargunv.maplibrecompose.demoapp.Demo
+import dev.sargunv.maplibrecompose.demoapp.DemoMapControls
+import dev.sargunv.maplibrecompose.demoapp.DemoOrnamentSettings
+import dev.sargunv.maplibrecompose.demoapp.DemoScaffold
+import dev.sargunv.maplibrecompose.demoapp.generated.Res
+import io.github.dellisd.spatialk.geojson.Position
+
+private val NEW_YORK = Position(latitude = 40.744, longitude = -73.981)
+
+object LocalTilesDemo : Demo {
+  // TODO spin this into a full local tiles demo
```

Comment:
This feature implements to programmatically add raster sources, you could just as well rename this demo to RasterTilesDemo and be done.
But if you want to have a local (raster) tiles demo, maybe ask Vichy97 to contribute some open data set?

Reply:
Yeah right now it's just remote raster tiles, but I want to demo using this for local tiles (as I think that'd be the primary use case for programmatically configuring the tile URL)